app/src/app/api/v0/auth/register/route.ts

```diff
@@ -27,6 +31,22 @@ export const POST = apiHandler(async (req: Request) => {
     await user.addCity(inventory?.cityId);
   }
 
+  // Send email to user
+  const host = process.env.HOST ?? "http://localhost:3000";
+  const sendInvite = await sendEmail({
+    to: body.email!,
+    subject: "City Catalyst - User Registration",
+    html: render(
+      ConfirmRegistrationTemplate({
+        url: `${host}/dashboard`,
```

Comment:
Are we sure the HOST env var includes the `https://` protocol prefix?

Reply:
Also in a future PR, this link needs to be changed to include a verification code that is generated when the email is sent and saved in the database (to compare to the value from the link the user clicks in the email they receive).